mixed-content/generic/worker.js

```diff
@@ -0,0 +1,7 @@
+try {
+  // Works on same origin.
+  postMessage('done');
+} catch(err) {
+  // Works cross-origin.
+  postMessage('done', '*');
```

Comment:
Sorry, I wasn't thinking: how do you ever have a cross-origin case here? Dedicated (and shared) workers are same-origin only, aren't they?